tests/ci/common_fuzz.sh

```diff
@@ -43,22 +43,25 @@ function put_metric {
 function run_fuzz_test {
   SHARED_CORPUS="${CORPUS_ROOT}/shared_corpus/${FUZZ_NAME}/shared_corpus"
   FUZZ_TEST_ROOT="${ALL_RUN_ROOT}/${FUZZ_NAME}"
+  LOCAL_SHARED_CORPUS_ROOT="${FUZZ_TEST_ROOT}/local_shared_corpus"
   FUZZ_TEST_CORPUS="${FUZZ_TEST_ROOT}/run_corpus"
   ARTIFACTS_FOLDER="${FUZZ_TEST_ROOT}/artifacts"
   FUZZ_RUN_LOGS="${FUZZ_TEST_ROOT}/logs"
   SUMMARY_LOG="${FUZZ_RUN_LOGS}/summary.log"
   mkdir -p "$SHARED_CORPUS" "$FUZZ_TEST_ROOT" "$FUZZ_TEST_CORPUS" "$ARTIFACTS_FOLDER" "$FUZZ_RUN_LOGS"
 
+  # To avoid having each libfuzzer thread read from the shared corpus copy it to the local CodeBuild directory one time
+  cp -r "$SHARED_CORPUS" "$LOCAL_SHARED_CORPUS_ROOT"
 
-    # Calculate starting metrics and post to CloudWatch
-  ORIGINAL_SHARED_CORPUS_FILE_COUNT=$(find "$SHARED_CORPUS" -type f | wc -l)
+  # Calculate starting metrics and post to CloudWatch
+  ORIGINAL_SHARED_CORPUS_FILE_COUNT=$(find "$LOCAL_SHARED_CORPUS_ROOT" -type f | wc -l)
   put_metric_count --metric-name SharedCorpusFileCount --value "$ORIGINAL_SHARED_CORPUS_FILE_COUNT" --dimensions "FuzzTest=$FUZZ_NAME"
 
   # Perform the actual fuzzing!
   # Step 1 run each fuzz test for the determined time. This will use the existing shared corpus (in EFS) and any files
   # checked into the GitHub corpus. This runs the fuzzer with three folders: the first folder is where new inputs will
-  # go (FUZZ_TEST_CORPUS), all other folders will be used as input (SHARED_CORPUS and SRC_CORPUS). It will write new
-  # files to the temporary run corpus.
+  # go (FUZZ_TEST_CORPUS), all other folders will be used as input ($LOCAL_SHARED_CORPUS_ROOT and SRC_CORPUS). It will
+  # write new files to the temporary run corpus.
```

Comment:
Is "temporary run corpus" = FUZZ_TEST_CORPUS?

Reply:
Yes, let me make these names more consistent.